static/app/views/replays/replayTable/tableCell.tsx

```diff
@@ -634,7 +638,7 @@ const Item = styled('div')<{isArchived?: boolean; isWidget?: boolean}>`
       ? `padding: ${space(0.75)} ${space(1.5)} ${space(1.5)} ${space(1.5)};`
       : `padding: ${space(1.5)};`};
   ${p => (p.isArchived ? 'opacity: 0.5;' : '')};
-  overflow: scroll;
+  ${p => (p.isReplayCell ? 'overflow: scroll;' : '')};
```

Comment:
```suggestion
  ${p => (p.isReplayCell ? 'overflow: auto;' : '')};
```

not sure about this, but `scroll` might mean scrollbars appear all the time for folks, while auto toggles them as needed

Reply:
`auto` works!